.kokoro/downstream-client-library-check.sh

```diff
@@ -59,7 +62,5 @@ if [[ $CLIENT_LIBRARY == "bigtable" ]]; then
   popd
 fi
 
-mvn verify install -B -V -ntp -fae \
```

Comment:
"test" type in `build.sh` seems to have the following command:
```
retry_with_backoff 3 10 \
  mvn install -B -V -ntp \
    -DskipTests=true \
    -Dclirr.skip=true \
    -Denforcer.skip=true \
    -Dmaven.javadoc.skip=true \
    -Dgcloud.download.skip=true \
    -T 1C

mvn test -B -ntp -Dclirr.skip=true -Denforcer.skip=true
```
which I'm not sure is equivalent to what you're replacing here.

Reply:
Ah thanks for catching this. I was trying to emulate the changes we made in gax-java but just realized that this line was recently reverted back to its original form in https://github.com/googleapis/gax-java/commit/c5eb06eec5d2cde85d37b096418e873c1bd83470. Sorry about the confusion.

Reply:
An update: it looks like updating that line to its original form is causing the native image compilation to be skipped for graal tests.